static/styles/recent_topics.css

```diff
@@ -131,6 +144,18 @@
             align-self: center;
         }
 
+        .unread_count_pm {
+            margin-right: 46px;
+        }
+
+        .user_circle {
+            min-width: 7px;
+            height: 7px;
+            margin-left: 20px;
+            margin-right: -7px;
```

Comment:
Can you add a comment explaining the intent of this block?  Negative margins are generally a bad pattern, and I wonder if we could do something more declarative with flex or another modern CSS feature here.